### PR TITLE
2022.2

### PR DESCRIPTION
**Pull request by antgonza**
Updating to 2022.2

### CODE REVIEW

File: qp_qiime2/__init__.py [hunk 1]
```diff
@@ -155,6 +172,10 @@
             # this is an odd one, first encountered:
             # feature-classifier fit-classifier-naive-bayes
             if ename == element.NOVALUE:
+                # As of qiime2-2022.2 nothing is filtered here, so let's raise
+                # an error so we can catch and review if this happens in the
+                # future
+                raise ValueError(f"[REVIEW] {pname} {mid} due to {ename}")
                 add_method = False
```
Comment (charles-cowart) [inline qp_qiime2/__init__.py]: if you raise ValueError here, add_method will never be set to False. Is this desired?

**Review by charles-cowart — CHANGES_REQUESTED** [review]
@antgonza Just one comment.

File: qp_qiime2/__init__.py (hunk 1 above)
Comment (antgonza) [reply qp_qiime2/__init__.py]: yes, if this happens we want everything to stop and have someone look - cause it should never happen.

**Comment by charles-cowart** [comment]
Right, but you have a statement that can never be reached afterwards. Would you mind removing it, please? Thank you!

File: qp_qiime2/__init__.py (hunk 1 above)
Comment (antgonza) [reply qp_qiime2/__init__.py]: duh! Now, now I understand what you mean ... yes.

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**